apps/omg_watcher_info/lib/omg_watcher_info/db/block.ex

```diff
@@ -125,6 +125,23 @@ defmodule OMG.WatcherInfo.DB.Block do
     from(block in __MODULE__, select: count())
   end
 
+  @spec query_timestamps :: Ecto.Query.t()
+  defp query_timestamps do
+    from(block in __MODULE__, select: %{timestamp: block.timestamp})
+  end
+
+  @spec get_timestamps :: []
+  def get_timestamps() do
+    DB.Repo.all(query_timestamps())
+  end
+
+  @spec get_timestamps_last_24_hours :: []
+  def get_timestamps_last_24_hours() do
```

Comment:
Curious, could you please provide the SQL this code generates? Just asking because I'm wondering if it might make sense to change `block.timestamp` from a simple postgres `integer` to an actual `timestamp` object.

This makes our storage of datetime-like columns consistent across all tables/columns - a small thing, but nice to have. Another, possibly bigger benefit would be the ability to use some of postgres' built-in datetime functionality like `interval`, etc..